doc/config.rst

```diff
@@ -1,10 +1,10 @@
 Configuration
 =============
 
-The *Config* can be modified from the input script for full customization of
-the calculations. By default low level optimisations are performed at PBE-D3BJ/def2-SVP,
-optimisations at PBE0-D3BJ/def2-SVP and single points at PBE0-D3BJ/def2-TZVP in
-ORCA if it is available.
+Configuration is handled with :code:`ade.Config` can be modified from the
```

Comment:
I don't think this sentence makes sense?

Reply:
you make a compelling point